### PR TITLE
First deliverable

### PR DESCRIPTION
**Pull request by memije**
Contains:
- Mini challenge 1
- Mini challenge 2

Overridden base repo in favor of a fresh new `create-react-app`.

### CODE REVIEW

File: src/components/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+const Button = styled.button`
+  font-size: 1em;
+  margin: 1em;
+  padding: 0.25em 1em;
+  border-radius: 3px;
+
+  background: ${(props) => (props.primary ? "#ff2e63" : "transparent")};
```
Comment (jonathanperazawl) [inline src/components/Button/index.js]: You can create a separate file to handle colors, it is usually recommended to create a `theme.js` file and inside it you can create a colors object:

```javascript
// theme.js
export const colors = {
  primary: '#ff2e63',
  white: '#fff',
  ...
}
```

File: src/components/Grid/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import Card from "../Card";
+
+const Grid = ({ data }) => {
+  return (
+    <div className="grid grid-cols-1 lg:grid-cols-4 md:grid-cols-3 gap-6">
+      {data.items.map(({ id, snippet: { title, description, thumbnails } }) => (
```
Comment (jonathanperazawl) [inline src/components/Grid/index.js:8]: +1 Good destructuring 🚀

**Review by jonathanperazawl — COMMENTED** [review]
Works fine, 

Thank you for this PR, I added small comments but this is a good deliverable without major comments, just as a recommendation on things you can improve:

* Responsive Design for the navigation bar component
* Adding padding to the container wrapper for videos

Also great job with the GitHub actions checks!

File: src/components/Button/index.js (hunk 1 above)
Comment (memije) [reply src/components/Button/index.js]: I had this very question 🤯 Thanks a lot

File: src/components/Grid/index.js (hunk 2 above)
Comment (memije) [reply src/components/Grid/index.js:8]: Thanks! 🙌

**Comment by memije** [comment]
@jonathanperazawl fixed comments 🚀

**Pull request closed without merging**